Header header 
nav_msgs/MapMetaData info
duration time_step
int16 entity
int8 n_steps
float32[] data
